cypress/integration/pages/testsForAllPages.js

```diff
@@ -53,11 +54,113 @@ const testsForAllPages = ({ service, pageType }) => {
           );
         });
 
-        it('should have the correct shared social media metadata', () => {
-          cy.checkSharedSocialmediaMetadata({
-            fbAdmins: '100004154058350',
-            appID: '1609039196070050',
+        it('should have the correct shared metadata', () => {
+          cy.get('head').within(() => {
+            cy.get('meta[name="fb:admins"]').should(
+              'have.attr',
+              'content',
+              '100004154058350',
+            );
+            cy.get('meta[name="fb:app_id"]').should(
+              'have.attr',
+              'content',
+              '1609039196070050',
+            );
+            cy.get('meta[name="og:image"]').should(
+              'have.attr',
+              'content',
+              appConfig[service].defaultImage,
+            );
+            cy.get('meta[name="og:image:alt"]').should(
+              'have.attr',
+              'content',
+              appConfig[service].defaultImageAltText,
+            );
+            cy.get('meta[name="og:locale"]').should(
+              'have.attr',
+              'content',
+              appConfig[service].locale,
+            );
+            cy.get('meta[name="og:site_name"]').should(
+              'have.attr',
+              'content',
+              appConfig[service].defaultImageAltText,
+            );
+            cy.get('meta[name="og:type"]').should(
+              'have.attr',
+              'content',
+              pageType === 'articles' ? 'article' : 'website',
+            );
+            cy.get('meta[name="og:url"]').should(
+              'have.attr',
+              'content',
+              `https://www.bbc.com${config[service].pageTypes[pageType].path}`,
+            );
+            cy.get('meta[name="og:site_name"]').should(
+              'have.attr',
+              'content',
+              appConfig[service].brandName,
+            );
+            cy.get('meta[name="twitter:card"]').should(
+              'have.attr',
+              'content',
+              'summary_large_image',
+            );
+            cy.get('meta[name="twitter:creator"]').should(
+              'have.attr',
+              'content',
+              appConfig[service].twitterCreator,
+            );
+            cy.get('meta[name="twitter:image:alt"]').should(
+              'have.attr',
+              'content',
+              appConfig[service].defaultImageAltText,
+            );
+            cy.get('meta[name="twitter:image:src"]').should(
+              'have.attr',
+              'content',
+              appConfig[service].defaultImage,
+            );
+            cy.get('meta[name="twitter:site"]').should(
+              'have.attr',
+              'content',
+              appConfig[service].twitterSite,
+            );
           });
+          cy.request(`${config[service].pageTypes[pageType].path}.json`).then(
+            ({ body }) => {
+              const description =
```

Comment:
This is not null but "null" for naidheachdan, this is a type problem, an issue should be raised to avoid this.

The lack of data is not only problem in our local fixture data, but a source data problem in the test and live environments too!!! We should urgently get this fixed across all three locations, please ensure there's and issue for adding the fixture data in this repo - but please don't let this be a hackfix for the type problem mentioned above.

To check for yourself, go here: https://www.bbc.co.uk/naidheachdan inspect and search for og:description or twitter:description